src/components/Select.vue

```diff
@@ -85,7 +85,7 @@ export default {
   },
   computed: {
     activeOptionIndex() {
-      console.log('options', this.options)
+      // TODO: implement find for operator select, wait till the Radon Library is updated
```

Comment:
Can you open an issue explainning this?
```suggestion
      // TODO(#issue): implement find for operator select, wait till the Radon Library is updated
```